### ISSUE TITLE
Mobile - MAC user - Nav Bar visibility issue on some pages for MAC users

### ISSUE THREAD

**Issue by jel535s** [MidwestAccessCoalition/lockbox_rails#481]
## Context
As our users are often accessing the app on mobile devices, we did an audit for usability on #424
When viewed on a Firefox mobile browser on a 4.7 inch screen, the buttons on the nav bar have some text which is hidden.  As a MAC user, I want to be able to read the text on the nav bar even when the text is wrapping to a second line within the buttons.
Blocked by #423 - merged, no longer blocking.

## Process
Using a iPhone SE (4.7 inch screen) and using the Firefox mobile browser, access the lockbox app as a MAC user. When looking at a Support Request, or when looking at a single Partner's page, the nav bar has 4 options: All Activity, File a Support Request, Add Cash to Lockbox, and Manage Lockbox. Due to the small screen size, the text within the buttons wraps to a second line for several options. This second line of text is obscured and might make navigation for a new user difficult. See screenshot in Resources.
To better match the Invisio design, can we move these nav bar options to within the hamburger menu under Quick Actions?

## Acceptance Criteria/Definition of Done
Either re-size the nav bar to show all text, or move the nav bar options under the hamburger menu to make better use of our mobile real estate on screen.

## Resources
Current mobile view for a MAC user:
![Screenshot 2020-08-30 at 4 52 10 PM (1)](https://user-images.githubusercontent.com/53353956/92336246-29fc0700-f064-11ea-946d-3735c89f9f74.png)
Invisio design:
https://projects.invisionapp.com/share/9EMYOUG7ZBA#/screens/398800920

**Comment by jel535s** [comment]
I'm really not sure why, but I'm still seeing the nav bar in our prod instance, both when viewing details for a specific lockbox and when looking at a Support Request on my mobile  (iphone SE, 4.7 inch screen, browsers Firefox and Safari). Re-opening this ticket- @nsiwnf can you take a look as you're most familiar with this work? Screenshots linked below
The VERY weird part is that when I test these same scenarios in the demo instance, the nav bar is hidden!
Creating a separate ticket to add the items from the nav bar to the Quick Actions menu so that mobile users still have all functionality available to them- #559 

Resources
Demo Instance: Viewing details for a Partner (Safari)
https://drive.google.com/file/d/1vBm48iZZQHJ4BOcb9qmPi-QbHCfxZFS1/view?usp=sharing
Demo instance: Viewing a Support Request (Safari)
https://drive.google.com/file/d/1UoDndDSN5e6xEzw5tIhXAMyEDb4bUUcP/view?usp=sharing
Prod instance: Viewing details for a Partner (Firefox)
https://drive.google.com/file/d/17FG-D5cdzd0AEVgrvpELbS5eJxDb8Din/view?usp=sharing
Prod instance: Viewing details for a Partner (Safari)
https://drive.google.com/file/d/19GKNimHogDkJ_5WTc0uIERjlJAvyzcFz/view?usp=sharing
Prod instance: Viewing a Support Request (Safari)
https://drive.google.com/file/d/17hNeQ9gR39ZQlerxJpNHIdwOQzOBi-ZU/view?usp=sharing

**Comment by jel535s** [comment]
Update: @BintLopez has a working theory that the auto deploy to staging failed, and what was promoted to prod on Jan 10 didn’t have this code in it. She thinks deploying and then fixing the bug we have in staging will be the way to proceed, and is testing that theory now.

**Comment by jel535s** [comment]
To solve for the above issue, @BintLopez  is commenting out the code that hides PR532 in a new PR today, so that when it gets deployed it doesn’t introduce the same bug that’s in staging

(So 532 will be included in tonight's deploy, but she’ll introduce new code that comments it out and then THAT will get fixed when we add the stuff to the Quick Actions menu on #559)
